packages/x-data-grid-premium/src/DataGridPremium/useDataGridPremiumProps.ts

```diff
@@ -32,6 +55,15 @@ export const DATA_GRID_PREMIUM_PROPS_DEFAULT_VALUES: DataGridPremiumPropsWithDef
   },
 };
 
+const getDataGridPremiumDefaultProps: (
+  themedProps: GetDataGridPremiumPropsDefaultValues,
+) => DataGridPremiumPropsWithDefaultValue = (themedProps) => ({
+  ...DATA_GRID_PREMIUM_PROPS_DEFAULT_VALUES,
+  filterDebounceMs: themedProps.unstable_dataSource
+    ? 1000
```

Comment:
Why this change? The filtering feels very slow

Reply:
The rationale would be to call this as less frequently as possible because too frequent BE calls would not be great with the backend data. Should we lower this value?

Reply:
> Should we lower this value?

For sure! I would keep the default one (150ms) and see if this raises concerns.